### PR TITLE
chore(doc): add 1.8.0 changelog in the repo

### PR DESCRIPTION
**Pull request by mittachaitu**
Signed-off-by: mittachaitu <sai.chaithanya@mayadata.io>
This PR adds changelog for 1.8.0-RC-1 in the repo.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+v1.8.0-RC1/ 2020-03-05
```
Comment (pawanpraka1) [inline CHANGELOG.md]: add RC2 changelog also.

Comment (mynktl) [reply CHANGELOG.md]: AFAIK if there are no changes then we are not adding.

Comment (mittachaitu) [reply CHANGELOG.md]: Done As discussed in slack thread PTAL.

**Review by pawanpraka1 — APPROVED** [review]
looks good.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,19 @@
+v1.8.0-RC2/ 2020-03-13
```
Comment (pawanpraka1) [inline CHANGELOG.md]: 2020-03-12?

Comment (mittachaitu) [reply CHANGELOG.md]: Our docker hub reports it was built 4 days ago
https://hub.docker.com/layers/openebs/cstor-istgt/1.8.0-RC2/images/sha256-f7e09b9d698211fdacfd18daee3c5ec0fad08b2cbe92696011a07c029296b61f?context=explore

Comment (pawanpraka1) [reply CHANGELOG.md]: this is about the tag not about the image, So the date is when this tag was created and all the changes done in this tag.

**Review by pawanpraka1 — APPROVED** [review]
looks good.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,24 @@
+v1.8.0/ 2020-03-14
+===================
+
+**No changes since v1.8.0-RC2**
```
Comment (kmova) [inline CHANGELOG.md]: Include the changes done in 1.8.0 and all of its RC builds.

Comment (mittachaitu) [reply CHANGELOG.md]: Done

**Review by kmova — APPROVED** [review]

**Pull request merged**